content/en/guide/v10/components.md

```diff
@@ -87,10 +87,10 @@ In order to have the clock's time update every second, we need to know when `<Cl
 | `componentWillMount()`        | (deprecated) before the component gets mounted to the DOM
 | `componentDidMount()`         | after the component gets mounted to the DOM
 | `componentWillUnmount()`      | prior to removal from the DOM
-| `componentWillReceiveProps(nextProps, nextState)` | before new props get accepted _(deprecated)_
-| `getDerivedStateFromProps(nextProps)` | just before `shouldComponentUpdate`. Use with care.
-| `shouldComponentUpdate(nextProps, nextState)` | before `render()`. Return `false` to skip render
-| `componentWillUpdate(nextProps, nextState)` | before `render()` _(deprecated)_
+| `componentWillReceiveProps(nextProps, context)` | before new props get accepted _(deprecated)_
+| `getDerivedStateFromProps(nextProps, prevState)` | just before `shouldComponentUpdate`. Use with care.
```

Comment:
> Also wondering whether it would be helpful to specify the expected return value for `getDerivedStateFromProps`

Sure, maybe something like this?

```suggestion
| `getDerivedStateFromProps(nextProps, prevState)` | just before `shouldComponentUpdate`. Return object to update state or `null` to skip update. Use with care.
```